x-pack/platform/plugins/shared/embeddable_alerts_table/public/components/embeddable_alerts_table.tsx

```diff
@@ -0,0 +1,133 @@
+  RULE_TYPES_LOAD_ERROR_TITLE,
+} from '../translations';
+import type { EmbeddableAlertsTableQuery } from '../types';
+
+export interface EmbeddableAlertsTableProps {
+  id: string;
+  timeRange?: TimeRange;
+  solution?: RuleTypeSolution;
+  query?: EmbeddableAlertsTableQuery;
+  services: AlertsTableProps['services'];
+}
+
+/**
+ * Renders the AlertsTable based on the embeddable table config
+ */
+export const EmbeddableAlertsTable = ({
+  id,
+  timeRange,
+  solution,
+  query,
+  services,
+}: EmbeddableAlertsTableProps) => {
+  const {
+    data: ruleTypes,
+    isLoading: isLoadingRuleTypes,
+    isError: cannotLoadRuleTypes,
+  } = useGetInternalRuleTypesQuery({ http: services.http });
+  const ruleTypeIds = useMemo(
+    () => (!ruleTypes || !solution ? [] : getRuleTypeIdsForSolution(ruleTypes, solution)),
+    [ruleTypes, solution]
+  );
+  const timeRangeQuery: QueryDslQueryContainer | null = timeRange
+    ? {
+        bool: {
+          minimum_should_match: 1,
+          should: [
+            getTime(undefined, timeRange, {
+              fieldName: ALERT_TIME_RANGE,
+            })!.query,
+            getTime(undefined, timeRange, {
+              fieldName: TIMESTAMP,
+            })!.query,
+          ],
+        },
+      }
+    : null;
+  const filtersQuery = query?.filters ? alertsFiltersToEsQuery(query?.filters) : null;
+  const finalQuery = {
+    bool: {
+      must: [timeRangeQuery, filtersQuery].filter(Boolean) as QueryDslQueryContainer[],
+    },
+  };
+
+  if (isLoadingRuleTypes) {
+    // Using EuiLoadingChart instead of EuiLoadingSpinner to match the
+    // dashboard panel loading indicator
+    return (
+      <EuiFlexGroup alignItems="center" justifyContent="center">
+        <EuiFlexItem grow={false}>
+          <EuiLoadingChart size="l" />
+        </EuiFlexItem>
+      </EuiFlexGroup>
+    );
+  }
+
+  if (cannotLoadRuleTypes) {
+    // If rule types are not available, we cannot forward their ids to the table for authorization
+    return (
+      <EuiEmptyPrompt
+        title={<h2>{RULE_TYPES_LOAD_ERROR_TITLE}</h2>}
+        body={<p>{RULE_TYPES_LOAD_ERROR_DESCRIPTION}</p>}
+        color="danger"
+        iconType="error"
+      />
+    );
+  }
+
+  if (solution && !ruleTypeIds.length) {
+    return (
+      <EuiEmptyPrompt
+        title={<h2>{NO_AUTHORIZED_RULE_TYPE_PROMPT_TITLE}</h2>}
+        body={<p>{getSolutionRuleTypesAuthPromptBody(solution)}</p>}
+        iconType="securityApp"
+      />
+    );
+  }
+
+  return (
+    <AlertsTable
+      id={id}
+      ruleTypeIds={ruleTypeIds}
+      query={finalQuery}
+      showAlertStatusWithFlapping
+      renderActionsCell={AlertActionsCell}
+      toolbarVisibility={{
+        // Disabling the fullscreen toggle since it breaks in Dashboards
+        // and panels can be maximized on their own
+        showFullScreenSelector: false,
+      }}
+      emptyStateHeight="flex"
+      casesConfiguration={{
+        featureId: 'alerts',
```

Comment:
Is this value correct?

Reply:
I do not think we can support cases atm the way it is implemented. Let's remove `casesConfiguration` even if we do not show any actions atm.

Reply:
The cases bulk actions do show up and seem to work correctly though

Reply:
Interesting. You are right. Is it easy to disable them? If we remove `casesConfiguration`, would it be hidden? If yes, could we disable atm? If it is not easy to do, it is ok to keep them.